frontend/app/src/components/landingpages/Team.svelte

```diff
@@ -26,7 +26,7 @@
             imageUrl={"/assets/team/steffen2.png"}
```

Comment:
I think the photo ios different too

Reply:
It is but I have strategically ignored the photo because I've already done it and it's the wrong shape.